versioned/persist/inmem/src/main/java/org/projectnessie/versioned/persist/inmem/InmemoryStore.java

```diff
@@ -49,7 +49,17 @@ public void configure(InmemoryConfig config) {}
   public void initialize() {}
 
   @Override
-  public void close() {}
+  public void close() {
+    globalStateLog.clear();
```

Comment:
Will GC not reclaim them anyway after the Store itself is no longer referenced?

Reply:
maybe - maybe not - depends on the tests being run

Reply:
Do you mean the `dispose` may be invoked, but the Store object still referenced after that?

Reply:
I mean that the store may survive because it's still referenced - the _contents_ of the store not cleaned up